README.md

```diff
@@ -2,11 +2,13 @@
 
 | Travis | Doc
 | --- | --- |
-| [![Build Status](https://travis-ci.org/lucidsoftware/rules_twirl.svg)](https://travis-ci.org/lucidsoftware/rules_twirl) | [Stardoc](http://lucidsoftware.github.io/rules_twirl/) |
+| [![Build Status](https://api.travis-ci.org/lucidsoftware/rules_play_routes.svg?branch=master)](https://travis-ci.org/lucidsoftware/rules_twirl) | [Stardoc](docs/stardoc/twirl.md) |
```

Comment:
The svg should be `https://api.travis-ci.org/lucidsoftware/rules_twirl.svg?branch=master`